src/handlers/guild.ts

```diff
@@ -622,3 +622,100 @@ export function getGuild(guildID: string, counts = true) {
     { with_counts: counts },
   ) as Promise<UpdateGuildPayload>;
 }
+
+/**
+ * Returns the guild template if it exists
+ * @param guildID The ID of the guild
+ * @param code The code of the template to get
+ */
+export function getGuildTemplate(guildID: string, code: string) {
+  const endpoint = `${endpoints.GUILD_TEMPLATES(guildID)}/${code}`;
+  return RequestManager.get(endpoint);
```

Comment:
I would just move the endpoint inside the function itself. Creating a variable is actually quite expensive function for 2 reasons.

1. its a much bigger performance issue than u realize. For example, i once ran a test 
and creating a var was slower by a pretty large factor.
2. This makes it so that its taking up memory until garbage collector or some part of v8 will clean this up.

You can avoid that entirely and move this directly inside the function